processor/resourcedetectionprocessor/internal/azure/aks/metadata.yaml

```
@@ -10,4 +10,8 @@ resource_attributes:
   cloud.platform:
     description: The cloud.platform
     type: string
-    enabled: true
\ No newline at end of file
+    enabled: true
+  k8s.cluster.name:
+    description: The k8s.cluster.name parsed from the Azure Instance Metadata Service's infrastructure resource group field
+    type: string
+    enabled: true
```

```suggestion
    enabled: false
```
I think we should keep it disabled by default.